packages/phishing-controller/src/PhishingDetector.test.ts

```diff
@@ -0,0 +1,979 @@
+            tolerance: 2,
+            version: 1,
+          },
+          {
+            allowlist: ['allowed-by-both.com'],
+            blocklist: [],
+            fuzzylist: [],
+            name: 'second',
+            tolerance: 2,
+            version: 1,
+          },
+        ],
+        async ({ detector }) => {
+          const { result, type, name } = detector.check('allowed-by-both.com');
+
+          expect(result).toBe(false);
+          expect(type).toBe('allowlist');
+          expect(name).toBe('first');
+        },
+      );
+    });
+
+    it('allows a domain when it is in the first fuzzylist and the second allowlist', async () => {
+      await withPhishingDetector(
+        [
+          {
+            allowlist: [],
+            blocklist: [],
+            fuzzylist: ['fuzzy-first-allowed-second.com'],
+            name: 'first',
+            tolerance: 2,
+            version: 1,
+          },
+          {
+            allowlist: ['fuzzy-first-allowed-second.com'],
+            blocklist: [],
+            fuzzylist: [],
+            name: 'second',
+            tolerance: 2,
+            version: 1,
+          },
+        ],
+        async ({ detector }) => {
+          const { result, type, name } = detector.check(
+            'fuzzy-first-allowed-second.com',
+          );
+
+          expect(result).toBe(false);
+          expect(type).toBe('allowlist');
+          expect(name).toBe('second');
+        },
+      );
+    });
+
+    it('allows a domain when it is in the first allowlist and the second fuzzylist', async () => {
+      await withPhishingDetector(
+        [
+          {
+            allowlist: ['allowed-first-fuzzy-second.com'],
+            blocklist: [],
+            fuzzylist: [],
+            name: 'first',
+            tolerance: 2,
+            version: 1,
+          },
+          {
+            allowlist: [],
+            blocklist: [],
+            fuzzylist: ['allowed-first-fuzzy-second.com'],
+            name: 'second',
+            tolerance: 2,
+            version: 1,
+          },
+        ],
+        async ({ detector }) => {
+          const { result, type, name } = detector.check(
+            'allowed-first-fuzzy-second.com',
+          );
+
+          expect(result).toBe(false);
+          expect(type).toBe('allowlist');
+          expect(name).toBe('first');
+        },
+      );
+    });
+
+    it('blocks a blocklisted domain when it ends with a dot', async () => {
+      await withPhishingDetector(
+        [
+          {
+            allowlist: [],
+            blocklist: ['blocked.com'],
+            fuzzylist: [],
+            name: 'first',
+            tolerance: 2,
+            version: 1,
+          },
+        ],
+        async ({ detector }) => {
+          const { result, type } = detector.check('blocked.com.');
+
+          expect(result).toBe(true);
+          expect(type).toBe('blocklist');
+        },
+      );
+    });
+
+    describe('with legacy config', () => {
```

Comment:
Similar as a previous comment. All of the above tests seem to pertain to the recommended config. What do you think about grouping them together in a separate `describe`? So we'd have two describes, one for recommended, one for legacy.

Reply:
Refactored in 38807df51f5e6e16a083e5398de7512a3a07d7f3